systemd/systemdtest/systemdtest.go

```diff
@@ -59,3 +60,14 @@ Type=simple
 	}
 	return output
 }
+
+func MockSystemdVersion(version int) (restore func()) {
```

Comment:
I think we typically put such functions in the respective package (systemd in this case), and then systemd.Version doesn't have to be exported?

Also, I would make it possible to mock the function with complete return value (int, error), not just hardcode one that returns (version, nil).

Reply:
Done! :-)